example/lib/main.dart

```diff
@@ -1,70 +1,39 @@
 import 'package:flutter/material.dart';
-import 'package:graphql_flutter/graphql_flutter.dart';
 
-import './mutations/mutations.dart' as mutations;
-import './queries/readRepositories.dart' as queries;
-
-const String YOUR_PERSONAL_ACCESS_TOKEN = '<YOUR_PERSONAL_ACCESS_TOKEN>';
+import 'bloc.dart' show Bloc, Repo;
```

Comment:
Instead of overhauling the example to use the bloc pattern, can you add it as a separate example, like `bloc_example`?

Reply:
will do